### ISSUE TITLE
Open project error

### ISSUE THREAD

**Issue by Vincent0706** [spyder-ide/spyder#16242]
### What steps reproduce the problem?
 Project-->Open project

### What is the expected output? What do you see instead?
I saw the spyder report the error like this
![image](https://user-images.githubusercontent.com/82581024/130027478-34766125-e502-432c-bddf-732e52cf417c.png)

### Paste Traceback/Error Below (if applicable)

```python-traceback

Traceback (most recent call last):
  File "D:\Program Files\Python39\lib\site-packages\spyder\plugins\projects\plugin.py", line 276, in <lambda>
    triggered=lambda v: self.open_project())
  File "D:\Program Files\Python39\lib\site-packages\spyder\plugins\projects\plugin.py", line 433, in open_project
    project_type_class = self._load_project_type_class(path)
  File "D:\Program Files\Python39\lib\site-packages\spyder\plugins\projects\plugin.py", line 864, in _load_project_type_class
    config.read(fpath)
  File "D:\Program Files\Python39\lib\configparser.py", line 697, in read
    self._read(fp, filename)
  File "D:\Program Files\Python39\lib\configparser.py", line 1017, in _read
    for lineno, line in enumerate(fp, start=1):
UnicodeDecodeError: 'gbk' codec can't decode byte 0xaa in position 178: illegal multibyte sequence

```

## Versions

* Spyder version: 5.1.1 None
* Python version: 3.9.5 64-bit
* Qt version: 5.12.10
* PyQt5 version: 5.12.3
* Operating System: Windows 10

**Comment by ccordoba12** [comment]
Hey @Vincent0706, thanks for reporting. This was caught incorrectly by our bot, sorry. I think this problem happens because you created a project in a directory with Chinese characters, so the solution for now is to avoid creating projects in those directories (sorry for the inconvenience).

**Comment by ccordoba12** [comment]
This is a duplicate of issue #15340 and we'll try to fix it in our next vesion (5.2.0), to be released at the end of September.

So I'm closing this issue in favor of that one.
